logger/logger.go

```diff
@@ -1,38 +1,14 @@
 package logger
```

Comment:
@wisdommatt We can remove this entire file now and move init of salt loger to server file only.

Reply:
Okay, i will do that now

Reply:
Log level can now move to https://github.com/odpf/guardian/blob/main/app/server.go#L42

Reply:
> Log level can now move to https://github.com/odpf/guardian/blob/main/app/server.go#L42

I just pushed a commit that resolves this